eventmesh-security-plugin/eventmesh-security-api/src/main/java/org/apache/eventmesh/api/acl/AclProperties.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.api.acl;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+public class AclProperties {
+
+    private String clientIp;
+    private String user;
+    private String pwd;
+    private String subsystem;
+    private String topic;
+    private Integer requestCode;
+    private String requestURI;
+    private String token;
+    private ConcurrentHashMap<String, Object> extendedField;
```

Comment:
Add this map field, but never set/get the map in this pull request, perhaps you need also expand the interface or method.

Reply:
Yes, I would resubmit the extension method for manipulating map